bokeh/plotting/_figure.py

```diff
@@ -648,6 +649,55 @@ def graph(self, node_source, edge_source, layout_provider, **kwargs):
         self.renderers.append(graph_renderer)
         return graph_renderer
 
+    def contour(self, x=None, y=None, z=None, levels=[], **visuals):
```

Comment:
I'll change the default value of `levels` to be `None`, as that it what it will be in the long run when you want the code to determine a sensible sequence of levels based on the data.

The types here now match those of `from_contour` as that is what is called to do all of the work.